examples/Radar_With_Surface_Obs.py

```diff
@@ -0,0 +1,185 @@
+"""
+========================================
+Plotting Radar With Surface Observations
+========================================
+
+This example plots radar and surface observations, using Pyart to read in the radar data,
+Matplotlib to plot it, and Metpy to read in the surface observations and plot the station
+plots.
+"""
+from datetime import datetime, timedelta
+
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
+import matplotlib
+import matplotlib.pyplot as plt
+from matplotlib import rcParams
+from metpy.calc import get_wind_components
+from metpy.plots import StationPlot
+from metpy.plots.wx_symbols import sky_cover
+from metpy.units import units
+import netCDF4
+import numpy as np
+import numpy.ma as ma
+import os.path
+import pyart
+from siphon.catalog import TDSCatalog
+from siphon.cdmr import Dataset
+from siphon.ncss import NCSS
+from siphon.radarserver import RadarServer
+import sys
+
+#################################################
+# Direct RadarServer to the correct link to the Amazon S3 server with the radar data.
+# The Amazon link only works for .edu domains, but the link below it will work for other
+# domains for data within the past two weeks or so.
+
+rs = RadarServer('http://thredds-aws.unidata.ucar.edu/thredds/radarServer/nexrad/level2/S3/')
+# rs = RadarServer('http://thredds.ucar.edu/thredds/radarServer/nexrad/level2/IDD/')
+
+#################################################
+# Use RadarServer and Pyart to pull in a radar object for the scan we want.
+
+query = rs.query()
+dt = datetime(2017, 6, 12, 22, 0) # Our specified time
```

Comment:
By using the fixed time, this will stop being able to pull surface data in a month or so. How about `datetime.utcnow()`.